h/models/token.py

```diff
@@ -55,6 +56,7 @@ class Token(Base, mixins.Timestamps):
         index=True,
         nullable=True,
     )
+    user: Mapped["User"] = sqlalchemy.orm.relationship(back_populates="tokens")
```

Comment:
Add a relationship so you can do `token.user` to get a token's user (works via the `token.user_id` foreign key relationship)